openapi/nexus.json

```diff
@@ -5342,14 +5366,28 @@
             "description": "During instance creation, create and attach disks",
             "type": "object",
             "properties": {
+              "block_size": {
+                "description": "size of blocks for this Disk. valid values are: 512, 2048, or 4096",
+                "allOf": [
+                  {
+                    "$ref": "#/components/schemas/ByteCount"
+                  }
+                ]
+              },
```

Comment:
I think it'd be nice if we had a `BlockSize` entry in the schema that look something like this: 

```json
{
  "BlockSize": {
    "description": "Valid block sizes for a disk",
    "type": "integer",
    "enum": [
      512,
      2048,
      4096
    ]
  }
}
```

It's not something we need to do in this PR, but it'd be nice to work towards. This would generate better types for us on the client and make it easier to automatically infer client side validation. As-is now we'd have to add special handling on the client for validation which is fine for where we are now but less desirable long term.

Reply:
No better time than the present :) done in ce7e8fd6